schema/rules.go

```diff
@@ -1,19 +1,27 @@
e, "and": true, "any": true, "args": true, "arguments": true, "array": true, "as": true, "as?": true, "ascending": true, "assert": true, "async": true, "await": true, "base": true, "bool": true, "boolean": true, "break": true, "by": true, "byte": true, "callable": true, "case": true, "catch": true, "chan": true, "char": true, "checked": true, "class": true, "clone": true, "const": true, "constructor": true, "continue": true, "debugger": true, "decimal": true, "declare": true, "def": true, "default": true, "defer": true, "del": true, "delegate": true, "delete": true, "descending": true, "die": true, "do": true, "double": true, "dynamic": true, "echo": true, "elif": true, "else": true, "elseif": true, "empty": true, "enddeclare": true, "endfor": true, "endforeach": true, "endif": true, "endswitch": true, "endwhile": true, "enum": true, "equals": true, "eval": true, "event": true, "except": true, "exception": true, "exit": true, "explicit": true, "export": true, "extends": true, "extern": true, "fallthrough": true, "false": true, "final": true, "finally": true, "fixed": true, "float": true, "fn": true, "for": true, "foreach": true, "from": true, "fun": true, "func": true, "function": true, "get": true, "global": true, "go": true, "goto": true, "group": true, "if": true, "implements": true, "implicit": true, "import": true, "in": true, "include": true, "include_once": true, "init": true, "instanceof": true, "insteadof": true, "int": true, "integer": true, "interface": true, "internal": true, "into": true, "is": true, "isset": true, "join": true, "lambda": true, "let": true, "list": true, "lock": true, "long": true, "managed": true, "map": true, "match": true, "module": true, "nameof": true, "namespace": true, "native": true, "new": true, "nint": true, "none": true, "nonlocal": true, "not": true, "notnull": true, "nuint": true, "null": true, "number": true, "object": true, "of": true, "on": true, "operator": true, "or": true, "orderby": true, "out": true, "override": true, "package": true, "params": true, "partial": true, "pass": true, "print": true, "private": true, "protected": true, "public": true, "raise": true, "range": true, "readonly": true, "record": true, "ref": true, "remove": true, "require": true, "require_once": true, "return": true, "sbyte": true, "sealed": true, "select": true, "set": true, "short": true, "sizeof": true, "stackalloc": true, "static": true, "strictfp": true, "string": true, "struct": true, "super": true, "switch": true, "symbol": true, "synchronized": true, "this": true, "throw": true, "throws": true, "trait": true, "transient": true, "true": true, "try": true, "type": true, "typealias": true, "typeof": true, "uint": true, "ulong": true, "unchecked": true, "unmanaged": true, "unsafe": true, "unset": true, "use": true, "ushort": true, "using": true, "val": true, "value": true, "var": true, "virtual": true, "void": true, "volatile": true, "when": true, "where": true, "while": true, "with": true, "xor": true, "yield": true}
```

Comment:
Looks like very long line. Can we break it for readability?
Not word per line, but by some rule:
  * 80-100 characters per line
  * Letter per line
  * 5-10 words per line
  * Line per language

Reply:
Also, consider using set.New() our lib object from `"github.com/tigrisdata/tigris/lib/set"`

Reply:
Thanks for pointing to set API.